MAUI/Masked-Entry/hiding-prompt-characters.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: Hiding prompt characters in .NET MAUI Masked Entry control | Syncfusion
+Description: Learn about hiding prompt characters support in Syncfusion .NET MAUI Masked Entry (SfMaskedEntry) control and more.
+platform: maui
+control: SfMaskedEntry
+documentation: ug
+---
+
+# Hiding Prompt Characters in .NET MAUI Masked Entry (SfMaskedEntry)
+
+When the HidePromptOnLeave property is set to true, prompt characters are ignored when control loses focus. Again, the prompt characters are restored when the control is focused.
```

Comment:
When the HidePromptOnLeave property is set to true, prompt characters are ignored when the control loses focus. Again, the prompt characters are restored when the control is focused.

Reply:
Content updated.